README.md

```diff
@@ -45,30 +105,24 @@ I recommend using
 a [cloudflare tunnel]('https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/get-started/create-remote-tunnel/'),
 or [ngrok]('https://ngrok.com/), or similar to expose your local server to the internet.
 
-### Login with Google
+#### Login with Google
 
 Create a project in Google Cloud Platform and enable the Google+ API. Then create OAuth 2.0 credentials and set the envs
 `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` with the values from the credentials.
 
 On the OAuth consent screen, set the authorized redirect URIs to `{your local env url}/accounts/google/login/callback/`
 
-### Once you have the envs set up, you can run the following commands:
-
-```sh
-python manage.py migrate
-whoami # copy the output
-python manage.py createsuperuser # paste the output as username, leave email empty, and set some password
-deactivate # only if you want to deactivate the virtualenv
-```
+#### Emails
 
 For email testing use https://mailtrap.io/
 
-### Running
+1. Create an account
+2. Get HOST, PORT, USERNAME and PASSWORD from Email Testing > Inboxes > SMTP
+
+### Run environment
 
 ```sh
-source venv/bin/activate # if not already activated from before
```

Comment:
No sacaría esto. Lo agregué porque tenés que hacerlo cada vez antes de arrancar el servidor. (necesitás estar dentro de virtual environment) Si venís acá en una semana o en un mes y querés ver cómo arrancar el servidor, no vas a leer todas las instrucciones de instalación, solo vas a bajar hasta "Run environment" (yo lo llamaría "Running" o "Running the app locally" o "Running the app") y vas a probar con `python manage.py runserver` y te vas a preguntar por qué no funciona.

Reply:
agregue "(venv)$" en todos los sh que necesitan correrse desde venv